tasktracker/config/models.py

```diff
@@ -6,20 +6,22 @@
 
 from tasktracker.models.model import Model
 
+BASE_DIR = Path().cwd().parent
```

Comment:
```suggestion
BASE_DIR = Path().cwd().parent.parent
```
As that was how it was originally setup?